scripts/stall-analyser.py

```diff
@@ -32,8 +32,8 @@
                     help='Skip common backtrace prefix terminated by one or more addresses greater or equal to the threshold (0=disabled)')
 parser.add_argument('-e', '--executable',
                     help='Decode addresses to lines using given executable')
-parser.add_argument('-f', '--full-addr2line', action='store_const', const=True, default=False,
-                    help='Decode full addr2line information, otherwise print concised report')
+parser.add_argument('-f', '--full-demangle', action='store_const', const=True, default=False,
```

Comment:
Now it's much easier to understand, but in both cases, the names are demangled, it's not really a question whether the function is "fully demangled" - it always is... Rather it's whether the parameter types are shown or not (if a function is overloaded, without them you won't know which one is referred to). It sounds to me like a name like "--function-params" or "--full-function-names" explains it better.

Reply:
ok, the former suggestion isn't accurate either since it's --function-params-types :)
I'll take your latter suggestion.  It sounds simpler and will force push a version with that fixed.